rustls/src/msgs/handshake.rs

```diff
@@ -1595,19 +1595,26 @@ impl Codec for ECParameters {
 #[derive(Debug, Clone)]
 pub struct DigitallySignedStruct {
     pub scheme: SignatureScheme,
+    #[deprecated(since = "0.20.7", note = "Use sig() accessor")]
     pub sig: PayloadU16,
 }
 
 impl DigitallySignedStruct {
+    #![allow(deprecated)]
     pub fn new(scheme: SignatureScheme, sig: Vec<u8>) -> Self {
         Self {
             scheme,
             sig: PayloadU16::new(sig),
         }
     }
+
+    pub fn sig(&self) -> &[u8] {
```

Comment:
Let's call this `signature()`? As I understand it the API guidelines would usually avoid abbreviations.